### ISSUE TITLE
Note not saved

### ISSUE THREAD

**Issue by igrlas** [Automattic/simplenote-electron#651]
I have lost few hours of work from Simplenote due to note not saving to the cloud storage.
I have expected to see a warning or something in case there was a problem saving the note, but instead everything looked fine when i closed the Simplenote app.

suggestions: 
1. check for note/connection health every so often, and report to user. maybe compare hash of the text in the note locally to the hash of the note on the server every minute or so and report any issue, this way user can at least manually save note to a .txt file before restarting the app.
2. give us a "force save to cloud" button. 
3. implement local history, if ever so crude one, better than nothing. 

#### Steps to reproduce
1. install fresh Simplenote on Ubuntu 17.10
2. log in to existing account
3. create new note titled noteOne in newly installed app
4. add text in the note noteOne
5. turn Ubuntu pc off and login to web editor
6. create random new note in webapp and add some text

#### What I expected
is to have note "noteOne" save to Simplenote backend, and to be displayed in web editor

#### What happened instead
there was only empty note in cloud, and on next startup, text in "noteOne" got deleted from local pc and lost forever
p.s. I have checked note history and nothing was there

#### OS version
Ubuntu mate 17.10
simple note app version 1.0.8

**Comment by roundhill** [comment]
Did you turn off your computer really quickly? The app auto saves after you stop typing for a few seconds.

**Comment by igrlas** [comment]
no, it was on for a while after i stopped using the app. And I was writing it on and off for couple of hours, so it should have been saved many times i guess.

**Comment by giosoft** [comment]
I haven't lost notes so far. but I'm a little bit scared. the desktop app gives no information about saving and syncing progress, success. igrlas's suggestions are perfect

**Comment by MorgothSauron** [comment]
sync progress and status should be available on all platform where Simplenote ca run.

I use 1Password and it gives sync progress in the notification area. usefull to chekc sync steps progress.
